src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ticketRuleModal.tsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import debounce from 'lodash/debounce';
+import * as queryString from 'query-string';
+
+import {addSuccessMessage} from 'app/actionCreators/indicator';
+import {ModalRenderProps} from 'app/actionCreators/modal';
+import ExternalLink from 'app/components/links/externalLink';
+import {t, tct} from 'app/locale';
+import space from 'app/styles/space';
+import {IssueConfigField} from 'app/types';
+import {IssueAlertRuleAction, IssueAlertRuleCondition} from 'app/types/alerts';
+import FieldFromConfig from 'app/views/settings/components/forms/fieldFromConfig';
+import Form from 'app/views/settings/components/forms/form';
+import {FormField} from 'app/views/settings/projectAlerts/issueEditor/ruleNode';
+
+type Props = ModalRenderProps & {
+  formFields: {[key: string]: any};
+  link?: string;
+  ticketType?: string;
+  instance?: IssueAlertRuleAction | IssueAlertRuleCondition;
+  onSubmitAction: (data: {[key: string]: string}) => void;
+  onPropertyChange: (rowIndex: number, name: string, value: string) => void;
+  index: number;
```

Comment:
```suggestion
  formFields: Record<string, any>;
  onSubmitAction: (data: Record<string, string>) => void;
  onPropertyChange: (rowIndex: number, name: string, value: string) => void;
  index: number;
  link?: string;
  ticketType?: string;
  instance?: IssueAlertRuleAction | IssueAlertRuleCondition;
```

Reply:
We should find a lint w/ autofix for this so you don't have to keep fixing all of these for me haha